frontend/src/modules/InvoiceModule/Forms/InvoiceForm.jsx

```diff
@@ -217,13 +217,7 @@ function LoadInvoiceForm({ subTotal = 0, current = null }) {
       <Divider dashed />
       <div style={{ position: 'relative', width: ' 100%', float: 'right' }}>
         <Row gutter={[12, -5]}>
-          <Col className="gutter-row" span={5}>
-            <Form.Item>
-              <Button type="primary" htmlType="submit" icon={<PlusOutlined />} block>
-                {translate('Save')}
-              </Button>
-            </Form.Item>
-          </Col>
+         
```

Comment:
When you delete a Col you should review the spacing provided by the gutter property in the Row.